include/vrv/layerelement.h

```diff
@@ -8,6 +8,7 @@
 #ifndef __VRV_LAYER_ELEMENT_H__
 #define __VRV_LAYER_ELEMENT_H__
 
+#include <set>
```

Comment:
Put system includes seprately. See https://github.com/rism-ch/verovio/blob/master/doc/guidelines.md#includes-and-forward-declarations